src/layouts/app/home-assistant.ts

```diff
@@ -43,19 +37,9 @@ export class HomeAssistantAppEl extends ext(HassBaseMixin(LitElement), [
   NotificationMixin,
   dialogManagerMixin,
 ]) {
-  private _route?: Route;
-  private _error?: boolean;
-  private _panelUrl?: string;
-
-  static get properties(): PropertyDeclarations {
-    return {
-      hass: {},
```

Comment:
Was it intention that `hass` and `routeData` are no longer properties?

Reply:
Yes. Hass is not defined in this file but inside base mixing. Route data was not used anymore